src/app/parcels/parcelsTable/ParcelsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { ErrorSecondaryText } from "@/app/errorStylingandMessages";
+import Modal from "@/components/Modal/Modal";
+import { Centerer, ContentDiv, OutsideDiv } from "@/components/Modal/ModalFormStyles";
+import { Suspense } from "react";
+import ExpandedParcelDetails from "../ExpandedParcelDetails";
+import ExpandedParcelDetailsFallback from "../ExpandedParcelDetailsFallback";
+import LinkButton from "@/components/Buttons/LinkButton";
+import Icon from "@/components/Icons/Icon";
+import { faBoxArchive } from "@fortawesome/free-solid-svg-icons";
+import { AppRouterInstance } from "next/dist/shared/lib/app-router-context.shared-runtime";
+import { useTheme } from "styled-components";
+import { SelectedClientDetails } from "./types";
+
+interface ParcelsModalProps {
+    router: AppRouterInstance;
+    modalIsOpen: boolean;
+    setModalIsOpen: (modalIsOpen: boolean) => void;
+    selectedParcelId: string | null;
+    selectedClientDetails: SelectedClientDetails | null;
+    modalErrorMessage: string | null;
+}
+
+const ParcelsModal: React.FC<ParcelsModalProps> = (props) => {
```

Comment:
purely a preference thing but I prefer props in a component argument to be unpacked just for ease of access
i.e
`= ({router, modalIsOpen, setModalIsOpen, ....})`